### ISSUE TITLE
"Fasterer was unable to process some files" with blank config/initializers/wrap_parameters.rb in Rails

### ISSUE THREAD

**Issue by mwiley** [DamirSvrtan/fasterer#4]
When I run

```
$ fasterer
```

in my Rails project, I get this error.

~~~
Fasterer was unable to process some files because the
internal parser is not able to read some characters.
Unprocessable files were:
-----------------------------------------------------
config/initializers/wrap_parameters.rb
~~~

The file has nothing but the default comments at the top. I would expect it to just skip this file.

**Comment by jackturnbull** [comment]
Getting the same for the following
```
Fasterer was unable to process some files because the
internal parser is not able to read some characters.
Unprocessable files were:
-----------------------------------------------------
config/initializers/backtrace_silencers.rb
config/initializers/inflections.rb
config/initializers/mime_types.rb
```
These files are just the default Rails 4.2, nothing too special about them (as far as I can tell)

**Comment by DamirSvrtan** [comment]
They're probably empty files, and error handling for empty files is something that was leftover from the previous parser I was using. I've released 0.1.2 so please check it out and reopen the issue if there are still some problems.
